.circleci/config.yml

```diff
@@ -255,6 +255,115 @@ jobs:
             MOCHA_FILE: integration-tests/test-results/junit/test-results-[hash].xml
           when: always
       - upload_artifacts
+  accessibility_test:
+    resource_class: medium+
+    docker:
+      - image: cimg/base:stable-20.04
+        auth:
+          username: dockstoretestuser
+          password: $DOCKERHUB_PASSWORD
+      - image: cimg/postgres:13.3
```

Comment:
wonder if this could be a variable to keep the various postgres image versions in sync

Reply:
I noticed that a few of the version numbers of different docker images are different. I have created a ticket to resolve this issue.

[SEAB-4703](https://ucsc-cgl.atlassian.net/browse/SEAB-4703)